collector/utils.go

```diff
@@ -230,11 +252,37 @@ func GetSettingCollection(client *rancher.Client) *rancher.SettingCollection {
 	return settings
 }
 
+func GetAllSettingData(client *rancher.Client) []rancher.Setting {
```

Comment:
All the functions below seem collector specific, why would `GetAllSettingData` be here instead of installation.go if that is only place that calls it ? And then `GetAllWorkloadData` and others are in their collector.

Reply:
Created `setting.go` file to define setting functions. Also, moved all `GetAll*` functions to their collectors.